frontend/src/components/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,76 @@
+import {
+  Avatar as ChakraAvatar,
+  AvatarBadge as ChakraAvatarBadge,
+  AvatarProps as ChakraAvatarProps,
+  Box,
+  ButtonProps,
+  Icon,
+  MenuItemProps,
+  useMultiStyleConfig,
+} from '@chakra-ui/react'
+
+import { BxsUser } from '~/assets/icons/BxsUser'
+
+import DropdownMenu from '../Menu'
+
+export interface AvatarProps extends ChakraAvatarProps {
+  hasNotification?: boolean
+}
+
+/*
+ * Utility to extract first letter of avatar name
+ * Override default getInitials Avatar method
+ */
+const getInitials = (fullName: string) => fullName[0]
+
+/*
+ * Avatar component
+ * May be wrapped in AvatarMenuButton
```

Comment:
I think avatar being in an dropdown menu is out of scope of this implementation. We can have a template component that uses Avatar, dropdown menu button, etc to compose a new AvatarMenuButton instead of having it be in this file

Reply:
thanks, shifted to template, see if that works